pkgs/development/python-modules/bap/default.nix

```diff
@@ -0,0 +1,20 @@
+{stdenv, buildPythonPackage, fetchFromGitHub, bap, requests}:
+
+buildPythonPackage rec {
+  name = "bap";
+  version = "1.1.0";
+  src = fetchFromGitHub {
+    owner = "BinaryAnalysisPlatform";
+    repo = "bap-python";
+    rev = "v${version}";
+    sha256 = "0wd46ksxscgb2dci69sbndzxs6drq5cahraqq42cdk114hkrsxs3";
+  };
```

Comment:
please add a `doCheck = False; # no tests` since we explicitly disable tests for python